SConstruct

```diff
@@ -378,8 +378,10 @@ if env['scalasca'] == 'kernels_2.x':
 #
 
 # enforce restrictive C/C++-Code
-env.Append(CFLAGS   = ['-Wall', '-Werror', '-ansi'],
-           CXXFLAGS = ['-Wall', '-Werror', '-ansi'])
+#env.Append(CFLAGS   = ['-Wall', '-Werror', '-ansi'],
```

Comment:
Two issues with that:
-  Please don't commit commented out code. That's what Git is for.
- While in principle I agree that this should be changed this is probably something that should be discussed separately.